node/node_test.go

```diff
@@ -256,23 +254,22 @@ func TestSpacemeshApp_Cmd(t *testing.T) {
 	r.Equal(config.JSONLogEncoder, app.Config.LOGGING.Encoder)
 }
 
-func marshalProto(t *testing.T, msg proto.Message) string {
-	var buf bytes.Buffer
-	var m jsonpb.Marshaler
-	require.NoError(t, m.Marshal(&buf, msg))
-	return buf.String()
+func marshalProto(t *testing.T, msg proto.Message) []byte {
+	buf, err := protojson.Marshal(msg)
+	require.NoError(t, err)
+	return buf
 }
 
-func callEndpoint(t *testing.T, endpoint, payload, address string) (string, int) {
+func callEndpoint(t *testing.T, endpoint string, payload []byte, address string) ([]byte, int) {
 	url := fmt.Sprintf("http://%s/%s", address, endpoint)
```

Comment:
The parameter order is odd to me: `endpoint`, `payload`, `address`. Maybe this could be changed to just `url`, `payload` similar to how it is in the `grpcserver_test.go` file?